packages/core/content-manager/admin/src/services/relations.ts

```diff
@@ -41,6 +41,9 @@ const relationsApi = contentManagerApi.injectEndpoints({
       }
     >({
       query: ({ model, id, targetField, params }) => {
+        if (id === undefined) {
+          return { url: '', method: 'GET', config: { params: {} } }; // Return a dummy URL to prevent the query
+        }
```

Comment:
I'm not 100% sure about this pattern. Isn't there a place we can invalidate some cache? Or where we could use isFetching to ensure we don't show stale data?

Reply:
The problem is before we were calling the GET api only when you have an ID, otherwise the API is not called, when you create a new entry with a Relation the GET api is called and the response coming from the API is assigned to the relations variable. Then when you click on the Back button, the API is not called and the value of relations remains the same...it is because of that I tried to mock the call to an API when you don't have an Id to respond with an empty array and then populate the relations variable with this array. unfortunately isFetching in this case has no effect

Reply:
the only way to do that is to invalidate the cache when you click on the BackButton